src/kanata.exe.manifest

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">
+  <assemblyIdentity name="kanata.exe" version="1.6.0.0" type="win32"/>
```

Comment:
Can we autogenerate this file and add to .gitignore, or put it in an already-ignored location, (does `target/` work, and does it make sense?). Automating having the version kept up-to-date would be good.

Sample of getting version in code:

https://github.com/jtroo/kanata/blob/c00fcadbfe197180b40cc0114b4620351505f66c/src/main.rs#L128

Reply:
Currently the manifest is just a placeholder since AHK tool doesn't work on a binary without one, so not sure it makes sense to generate it (but added it in the latest update)

But when you get a proper installer that can conditionally enable UIA (since it has some [limitations](https://www.autohotkey.com/docs/v1/Program.htm#Installer_uiAccess) which may be relevant from some users doing some interop with other apps, that installer can generate the proper versions/flags at runtime and embed if needed (and move to Program Files)

P.S.
Note that manifests for some reason require a 4-group version `n.n.n.n`, added some regexes for the common `-release-1` types to convert and a placeholder `0.0.0.0` on fails